db/seeds/hhg_weights.json

```diff
@@ -0,0 +1 @@
, "Clock (Grandfather)": 140, "Desk (Secretary)": 245, "Desk (Small)": 154, "Fireplace Equipment": 35, "Fish Tank": 105, "Footstool": 14, "Home Computer": 30, "Lamp (Floor/Pole)": 21, "Magazine Rack": 14, "Mirror Cartons/Pictures": 25, "Organ (Medium)": 420, "Organ (Small)": 140, "Piano (Baby Grand)": 490, "Piano (Parlor Grand)": 560, "Piano (Spinet )": 420, "Radio (Table)": 14, "Record Player": 14, "Rug (Large/Pad)": 70, "Rug (Small/Pad)": 21, "Sofa (2 Cushion)": 245, "Sofa (3 Cushion)": 315, "Sofa (4 Cushion)": 420, "Sofa (Section Per)": 210, "Speaker (Stereo)": 35, "Studio (Hidebed)": 350, "TV Big Screen": 100, "TV Combination": 175, "TV Console": 105, "TV Stand": 35, "TV Table Model": 70, "TV Tray Set": 21, "Table (Coffee/End)": 35, "Table (Dropleaf)": 84, "Telephone Stand": 35, "VCR": 35, "Video Game Set": 35, "Wine Rack": 35}, "Miscellaneous": {"Ash/Trash Can": 49, "Barbell Set (Small)": 35, "Basket (Clothes)": 35, "Bicycle": 35, "Bird Cage/Stand": 35, "Bowling Ball/Bag": 14, "Cabinet (File)": 140, "Car Carrier": 70, "Card Table": 7, "Chair (Folding)": 7, "Clothes Hamper": 35, "Computer and Accessories": 35, "Cot (Folding)": 70, "Desk (Office)": 210, "Dog House": 70, "Exercise Bike": 105, "Fan": 35, "Floor Polisher": 21, "Footlocker/Trunk": 35, "Golf Bag": 14, "Hall Tree": 14, "Heater (Gas/Electric)": 35, "Metal Shelves/Section": 35, "Moped": 105, "Motor (Electric)": 7, "Movie Screen": 7, "Movie/Slide Projector": 7, "Ping Pong Table": 140, "Plant Stand": 70, "Pool Table": 280, "Power Tools": 140, "Printer/Keyboard": 5, "Rowing Machine": 70, "Scooter": 35, "Sewing Machine": 70, "Skis/Poles": 21, "Sled": 14, "Sleeping Bag": 7, "Step Ladder": 35, "Suitcase": 35, "Table (Utility)": 35, "Tackle Box": 7, "Tool Chest": 70, "Treadmill": 50, "Tricycle": 35, "Typewriter": 56, "Vacuum Cleaner": 35, "Wagon (Child's)": 35, "Wardrobe Cartons/Clothes": 45, "Waste Paper Basket": 14, "Weight Bench": 105, "Work Bench": 140, "Work Mate": 70}, "Nursery": {"Baby Carriage": 70, "Basket (Plastic)": 21, "Bassinet": 35, "Bed (Youth)": 70, "Car Seat": 14, "Chair (Child's)": 21, "Changing Table": 70, "Chest": 91, "Chest (Toy)": 35, "Crib (Baby)": 70, "Doll House (Small)": 14, "Play House": 70, "Play Pen": 70, "Rug (Large/Pad)": 70, "Rug (Small/Pad)": 21, "Stroller": 35, "Table (Child's)": 35}, "Porch, Outdoor Furniture/Equipment": {"BBQ/Port Grill": 70, "Bird Bath": 35, "Camp Stove": 14, "Chair (Outdoor)": 70, "Child's Pool": 84, "Clothes Dry Rack": 35, "Garden Hose/Tools": 70, "Glider/Settee": 140, "Hand truck/Dolly": 14, "Ladder (Extension)": 70, "Lawn Edger": 35, "Lawn Mower (Hand)": 35, "Lawn Mower (Power)": 105, "Lawn Mower (Riding)": 245, "Leaf Sweeper": 35, "Outdoor Gym": 140, "Outdoor Slide": 70, "Outdoor Swings": 105, "Picnic Bench": 35, "Picnic Table": 140, "Rocker (Swing)": 105, "Roller (Lawn)": 105, "Sand Box": 70, "Saw Horse": 35, "Settee": 140, "Snow Blower": 105, "Spreader": 7, "TV Antenna": 35, "Table (Picnic)": 70, "Umbrella": 35, "Wheelbarrow": 56}}
```

Comment:
Is it customary to talk about refrigerators and freezers in terms of just CU? I think they are measured in cu. ft., right?

Reply:
Yeah, just CU doesn't make sense. This is taken directly from DPS. We should probably go through and make content edits, since there are formatting inconsistencies and other things that are confusing. cc @LillyMadigan

Reply:
Also, please pretty-print this file. I asked Jason for the same affordance on his PR. Having newlines and whitespace in the file make it much easier to read and makes small diffs much easier to visualize.